Marlin/Configuration_adv.h

```diff
@@ -138,7 +138,7 @@
 
 // Show Temperature ADC value
 // Enable for M105 to include ADC values read from temperature sensors.
-//#define SHOW_TEMP_ADC_VALUES
+#define SHOW_TEMP_ADC_VALUES
```

Comment:
Please don't modify the base configurations.

Reply:
Understood.